### PR TITLE
Adding Chaninverse os Package

### PR DESCRIPTION
**Pull request by jose-zamora**
# What changed
chainversOS

# How did you test the change?
added the feature flag
https://www.loom.com/share/35a3e50b1f75495f898c093da80008f1
- enabling: `window.showChainverseOS()`
- disabling: `window.removeChainverseOS()`

### CODE REVIEW

**Comment by liminal-dog** [comment]
Nice! 

Couple of questions:

1. What would you like us to review/test? 
2. What will happen when we accept the PR? For instance, will there be any changes to the existing search functionality hosted at `staging.chainverse.diamonds`

**Comment by jose-zamora** [comment]
hi @liminal-dog no actions required on your end. I added you just to let you know about this big progress integration. No worries 😉

File: src/pages/api/graphql.page.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { ApolloServer } from "apollo-server-micro";
 import Cors from "micro-cors";
 const cors = Cors();
 
-import { typeDefs } from "../../services/Apollo/typedefs";
 import neo4j from "neo4j-driver";
 import { Neo4jGraphQL } from "@neo4j/graphql";
+import { typeDefs } from "@chainverse/os";
```
Comment (clemp) [inline src/pages/api/graphql.page.ts]: Very cool that you were able to make the type definitions as an import separate from the frontend code. I followed along to find this import coming from the `chainverse-os` repository (https://github.com/DiamondDAO/chainverse-os/search?q=typedefs).

With this import pattern I imagine we can create something like a "Data Model Marketplace" with different custom types depending on the use case of the application being built; all the developer has to do is import the data model (i.e. type def) for their specific application.

File: src/pages/explorer/index.page.tsx [hunk 2]
```diff
@@ -6,31 +6,28 @@ import {
...
+} from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Layout } from '@/components/Layout';
+import { SearchIcon } from '@chakra-ui/icons';
+import { useSpring, a } from 'react-spring';
+import Router from 'next/router';
+import * as styles from './styles';
+import { SearchOS, SearchTypes } from '@chainverse/os';
+import { EntityTable } from '@/components/Explorer/EntityTable';
```
Comment (clemp) [inline src/pages/explorer/index.page.tsx:26]: Noting here that the `SearchOS` and `SearchTypes` are coming from the open source `chainverse-os` package, and the actual display of the data is coming from the `chainverse-frontend` code; exactly what we discussed for this sprint.

**Pull request merged**